site/components/search.tsx

```diff
@@ -125,6 +125,7 @@ export default function Search({ categories, brands }: SearchPropsType) {
                     >
                       <Link
                         href={{ pathname: getCategoryPath('', brand), query }}
+                        legacyBehavior
```

Comment:
I image these are because of the onclick needed on the a tag?

Reply:
Yes, because there was a warning if you put onClick on the <Link ...>